api/management/api.go

```diff
@@ -186,6 +187,11 @@ func ConvertToAPIUser(userKc kc.UserRepresentation) UserRepresentation {
 			counter, _ := strconv.Atoi(smsSent)
 			userRep.SmsSent = &counter
 		}
+
+		if m["trustid_groups"] != nil {
```

Comment:
I would suggest trustIDGroups (to respect the same format as the others)

Reply:
Done :-)